docs/source/usage/caravel/index.md

```diff
@@ -1544,7 +1544,7 @@ violations and 1 hold violation which is not Reg to Reg.
 ```
 
 The max Slew/Cap violations can be fixed the same way in
-[openroad-stapostpnr](openroad-stapostpnr). For the hold violation, it is in the
+[openroad-stapostpnr](#openroadstapostpnr). For the hold violation, it is in the
```

Comment:
Not the right reference

Reply:
I tried it, it worked, can you please confirm this is broken?